eng/pipelines/coreclr/jit-offbydefault-extra.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+pr: none
+
+schedules:
+- cron: "0 10 * * 6,0"
+  displayName: Sat and Sun at 2:00 AM (UTC-8:00)
```

Comment:
Is there any other pipelines running at the same time that would compete with this one?

Reply:
This kicks off at the same time as the other weekend runs, I can certainly change it...